api/rosetta/data_balance_integration_test.go

```diff
@@ -146,12 +146,12 @@ func TestAPI_BalanceHandlesErrors(t *testing.T) {
 	)
 
 	const (
-		invalidAddress = "0000000000000000" // valid 16-digit hex value but not a valid account ID
+		invalidAddress = "0000000000000000" // Valid 16-digit hex value but not a valid account ID
```

Comment:
I'm happy you made all of the changes to even single line comments to use upper case and periods a t the end. However, for these kind of inline comments, I will argue that it's the only case where they should stay lower case.